Manager/src/main/java/io/raspberrywallet/manager/bitcoin/Bitcoin.java

```diff
@@ -208,10 +215,10 @@ public void onSuccess(@Nullable Object result) {
                     @Override
                     protected void progress(double pct, int blocksSoFar, Date date) {
                         super.progress(pct, blocksSoFar, date);
-                        Logger.d("Progress " + pct + "%");
+                        Logger.d("Progress " + (int) Math.round(pct) + "%");
```

Comment:
does it work correctly? (doule to integer casting problem on frontend, that was causing progress bar jumping
we were talking about that, not sure if you remember